src/styles/main.scss

```diff
@@ -1,7 +1,13 @@
-@import "utils";
-@import "fonts";
-@import "typography";
-
-body {
-  background: $c-gray;
-}
+@import "../styles/utils";
+@import "../styles/header.scss";
```

Comment:
According to BEM, it is not recommended to create a separate .scss file for each block. Instead, consider grouping related blocks into fewer .scss files. This will make your code more maintainable and will reduce the amount of HTTP requests.